lib/ansible/modules/cloud/amazon/s3_bucket.py

```diff
@@ -133,6 +133,25 @@
     name: mydobucket
     s3_url: 'https://nyc3.digitaloceanspaces.com'
 
+# Create a bucket with AES256 encryption
+- s3_bucket:
+    name: mys3bucket
+    state: present
+    encryption: "AES256"
+
+# Create a bucket with aws:kms encryption, KMS key
+- s3_bucket:
+    name: mys3bucket
+    state: present
+    encryption: "aws:kms"
+    encryption_key_id: "arn:aws:kms:us-east-1:1234/5678example"
+
+# Create a bucket with aws:kms encryption, default key
+- s3_bucket:
+    name: mys3bucket
+    state: present
+    encryption: "aws:kms"
+    encryption_key_id:
```

Comment:
I was a little overzealous. Minor nit - I'd remove this since it's equivalent to omitting it, and I'd add a second task to check that changed == False the second time around. But looks great functionally.